resources/README.md

```diff
@@ -7,6 +7,21 @@ Resources are all components in Kyma that are available for local and cluster in
 Resources currently include, but are not limited to, the following:
 
 - Elements which are essential for the installation of `core` components in Kyma, such as certificates, users, and permissions
-- The `core` components required to run the Kyma
 - Examples of the use of specific components
 - Scripts for the installation of Helm, Istio deployment, as well as scripts for validating Pods, starting Kyma, and testing
+
+
+## Installation
```

Comment:
There are `service monitors` installed by various components and these require CRDs installed by prometheus operator. We want to leave these `service monitors` with the components, since components owners would take care of them.

Reply:
As we discussed we should create a follow-up task to investigate the way how we want to do that in the future. IMO it's strange that even when I disabling monitoring then some components are still created. You can have a problem on cluster where whole monitoring is preinstalled (including Prometheus) and AFIAK the prometheus is already preinstalled on Gardener, so with that modularization, we will still have a problem with the Kyma installation.

Reply:
Follow Up task: https://github.com/kyma-project/kyma/issues/1840.